src/main/java/org/hl7/davinci/priorauth/ClaimEndpoint.java

```diff
@@ -214,14 +222,30 @@ private IBaseResource processBundle(Bundle bundle) {
       bundleMap.put("resource", bundle);
       App.DB.write(Database.BUNDLE, bundleMap);
 
-      // Store the claim...
-      claim.setId(id);
-      Map<String, Object> claimMap = new HashMap<String, Object>();
-      claimMap.put("id", id);
-      claimMap.put("patient", patient);
-      claimMap.put("status", Database.getStatusFromResource(claim));
-      claimMap.put("resource", claim);
-      App.DB.write(Database.CLAIM, claimMap);
+      // Store the claim items...
+      if (claim.hasItem()) {
+        if (related != null) {
+          // Update the items...
+          for (ItemComponent item : claim.getItem()) {
+            Map<String, Object> dataMap = new HashMap<String, Object>();
+            Map<String, Object> constraintMap = new HashMap<String, Object>();
+            constraintMap.put("id", related.getIdElement().asStringValue());
+            constraintMap.put("sequence", item.getSequence());
+            dataMap.put("id", id);
+            dataMap.put("status", ClaimStatus.CANCELLED.getDisplay().toLowerCase());
```

Comment:
This is where it defaults to update the item to be Cancelled. In the IG it references a profile for Item and includes the ItemCancelledFlag. However this doesn't exist in the Java Class so I'm not sure how we're supposed to see the contents. 
There are also no examples illustrating this.

Reply:
You have to look at the updated FHIR resources for the IG, sometimes there are changes made to them. In that case, you need to build new Java classes that inherit from the base r4 classes and make changes to them. In CRD we added to the name of the class and did something like DaVinciDeviceRequest, see the following: https://github.com/HL7-DaVinci/CRD/blob/master/resources/src/main/java/org/hl7/davinci/stu3/fhirresources/DaVinciDeviceRequest.java